ClosedXML_Tests/Excel/CalcEngine/FunctionsTests.cs

```diff
@@ -658,5 +659,131 @@ public void TestOmittedParameters()
+            Assert.AreEqual(123.45, actual);
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"1,234.56\")");
+            Assert.AreEqual(1234.56, actual);
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"+1\")");
+            Assert.AreEqual(1, actual);
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"+1.23\")");
+            Assert.AreEqual(1.23, actual);
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"-1\")");
+            Assert.AreEqual(-1, actual);
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"-1.23\")");
+            Assert.AreEqual(-1.23, actual);
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\".1\")");
+            Assert.AreEqual(0.1, actual);
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"1.234567890E+34\")");
+            Assert.AreEqual(1.234567890E+34, actual);
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\".99999999999999\")");
+            Assert.AreEqual(0.99999999999999, actual);
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"-.99999999999999\")");
+            Assert.AreEqual(-0.99999999999999, actual);
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"1,23,4\")");
+            Assert.AreEqual(1234, actual);
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"1,234,56\")");
+            Assert.AreEqual(123456, actual);
+        }
+
+        [Test]
+        public void NumberValueExceptions()
+        {
+            Object actual;
+
+            try
+            {
+                actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"123.45\", \".\", \".\")");
+                Assert.Fail();
+            }
+            catch (CellValueException ex)
+            {
+                Assert.AreEqual("CurrencyDecimalSeparator and CurrencyGroupSeparator have to be different.", ex.Message);
+            }
+
+            try
+            {
+                XLWorkbook.EvaluateExpr("NUMBERVALUE(\"1.234.56\")");
+                Assert.Fail();
+            }
+            catch (CellValueException ex)
+            {
+                Assert.AreEqual("Input string was not in a correct format.", ex.Message);
+            }
+
+            try
+            {
+                XLWorkbook.EvaluateExpr("NUMBERVALUE(\"1.234,56\")");
+                Assert.Fail();
+            }
+            catch (CellValueException ex)
+            {
+                Assert.AreEqual("Input string was not in a correct format.", ex.Message);
+            }
+
+            try
+            {
+                XLWorkbook.EvaluateExpr("NUMBERVALUE(\"12;34\")");
+                Assert.Fail();
+            }
+            catch (CellValueException ex)
+            {
+                Assert.AreEqual("Input string was not in a correct format.", ex.Message);
+            }
+
+            //try
+            //{
+            //    XLWorkbook.EvaluateExpr("NUMBERVALUE(\"10/2\")"); //Excel returns 43141 - dont know why
```

Comment:
Excel treats this as February, 10th of the current year.

Reply:
I think we should accept to compromise on certain edge cases that are too difficult to implement straight away.

Reply:
I guess you're right.